components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -126,7 +126,7 @@ private static ImmutableMap<String, Integer> createChannelPriorities() {
   // -- Fields --
 
   /** Offsets to memory blocks, paired with their corresponding description. */
-  private List<Long> offsets;
+  private List<MemoryBlock> offsets;
```

Comment:
Should this internal variable be renamed as 

```
private List<MemoryBlock> blocks;
```
or

```
private List<MemoryBlock> memoryBlocks;
```
?

This means updating a few tens of references across the reader but arguably most of these lines are already updated by this PR.

Reply:
Done in 18ed364dcc, along with a few other places that reference offsets where (memory) blocks is now more appropriate.